src/schema/metadata/IncludedEyeMovementEvents.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: IncludedEyeMovementEvents
+description: |
+  List of included events with message specifications.
+  For example, if fixation markers from the EyeLink are included add:
+  `[[Start of fixation: “SFIX”], [End of fixation: “EFIX”]]`.
+  If no events are included write `None`.
+anyOf:
+  - type: object
```

Comment:
says object but the example in the definition above and the main document show array of arrays